public/locations-list-map.js

```diff
@@ -507,17 +541,18 @@ function googleMapsUri(address) {
 
 function getEntryEl(entry) {
   if (!entry.domElem) {
-    entry.domElem = ce('div', 'location');
-    ac(entry.domElem, ce('h4', null, ctn(entry.name)));
+    entry.domElem = ce('div', 'location py-3');
+    const header = ce('div', 'd-flex');
+    const headerHospitalInfo = ce('div', 'flex-grow-1');
+    const headerOrgType = ce('div', 'flex-grow-1 d-flex justify-content-end text-pink');
```

Comment:
Hopefully there's a better way to do the layout than hard-coding these specific classes?

Reply:
I like Nick's idea of handlebars templates rather than building it in js like this. I think I'd also lean towards refactoring to that after v2 is out, thoughts?